Automattic/newspack-plugin
# Unified Analytics Wizard

### All Submissions:






### Changes proposed in this Pull Request:

Converts the Google Analytics Wizard into a "unified" wizard similar to most others. 

<img width="1603" alt="Screen Shot 2019-09-12 at 8 22 21 AM" src="https://user-images.githubusercontent.com/1477002/64783747-910cdb80-d536-11e9-816e-18be800a0785.png">

### How to test the changes in this Pull Request:

1. Check out this branch, execute `npm run build:webpack`
2. Navigate to Analytics Wizard.
3. Verify wizard loads and resembles screenshot. 
4. Verify clicking `View` on Google Analytics `ActionCard` executes a handoff to Site Kit's Analytics dashboard.

### Other information:
